custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py

```diff
@@ -241,7 +250,7 @@ def chat(self, completion_create_params):
         else:
 
             def generator():
-                message_content = ""
+                message_content = []
```

Comment:
When this is a list, we need to change the below `message_content +=` to `message_content.append` because the current suggestion is appending content as individual chars to the list